<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>getKey</class>
 <widget class="QDialog" name="getKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;</string>
  </property>
  <widget class="QPushButton" name="next_Button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(33, 32, 66);</string>
   </property>
   <property name="text">
    <string>Закончить регистрацию</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>821</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="name_label">
      <property name="styleSheet">
       <string notr="true">color: transfert;</string>
      </property>
      <property name="text">
       <string>lc,lc,dl,s</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Сгенерировать ключ</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="key_lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px solid rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 600 18pt &quot;Segoe UI Variable Display Semib&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="key_label">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 30pt &quot;Segoe UI Variable Display Semib&quot;;</string>
      </property>
      <property name="text">
       <string>   Ключ</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="errorkey_label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="tockenchoose_Button">
        <property name="styleSheet">
         <string notr="true">background-color: transfert;
color:transfert
</string>
        </property>
        <property name="text">
         <string>Нажмите, чтобы выбрать носитель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="keychoose_label">
        <property name="styleSheet">
         <string notr="true">background-color: transfert;
color: transfert</string>
        </property>
        <property name="text">
         <string>Носитель выбран:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="wherekey_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>использовать токен</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>461</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../build/img/img.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="regist_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 15pt &quot;Segoe UI Variable Display Semib&quot;;</string>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="psw_label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: transfert;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>regist_label</zorder>
  <zorder>background_label</zorder>
  <zorder>psw_label</zorder>
  <zorder>next_Button</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>back_Button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
